l10n_ch_pain_credit_transfer/data/pain.001.001.03.ch.02.xsd

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This schema is coming from an external source so please do not modify it.
If linters have any problem with that, just exclude it from files to be checked with something like https://github.com/OCA/l10n-italy/blob/00f0eb7417026f8b1a6eea3a0f8a8c299938eb74/.pre-commit-config.yaml#L17-L18

Reply:
@SimoRubi thanks